internal/storage/v2/clickhouse/tracestore/reader_test.go

```diff
@@ -518,15 +543,16 @@ func TestFindTraces(t *testing.T) {
 func TestFindTraceIDs(t *testing.T) {
 	driver := &testDriver{
 		t: t,
-		expectedQuery: `SELECT DISTINCT trace_id FROM spans WHERE 1=1 ` +
-			`AND service_name = ? AND name = ? ` +
-			`AND duration >= ? AND duration <= ? ` +
-			`LIMIT ?`,
-		rows: &testRows[string]{
-			data: []string{
-				"00000000000000000000000000000001",
-				"00000000000000000000000000000002",
-			},
+		expectedQuery: `
+SELECT DISTINCT
+    s.trace_id,
+    t.start,
+    t.end
+FROM spans s
+LEFT JOIN trace_id_timestamps t ON s.trace_id = t.trace_id
+WHERE 1=1 AND s.service_name = ? AND s.name = ? AND s.duration >= ? AND s.duration <= ? LIMIT ?`,
```

Comment:
How is it that you don't have time bounds on the search? Time range (lookback) is the mandatory query condition